src/pages/api/event/[id]/index.ts

```diff
@@ -17,10 +17,10 @@ async function getEvent(
   req: NextApiRequest,
   res: NextApiResponse<EventResponse>
 ) {
-  const { id } = req.query;
+  const eventId = req.query.id as string;
```

Comment:
This could be a string array as you can pass multiple items

Reply:
@justinliangg do we want to deal with this now? or down the road when it becomes a problem. Happy to approve it and to merge it in otherwise

Reply:
@skyheat I’ve added some validation in a recent commit.

Reply:
oh ok. approved then